v21.1/create-changefeed.md

```diff
@@ -129,7 +127,33 @@ Below is a list of unsupported SQL types and values for Avro changefeeds:
     To avoid `NaN` or infinite values, add a [`CHECK` constraint](check.html) to prevent these values from being inserted into decimal columns.
     {{site.data.alerts.end}}
 
-- [`TIME`, `DATE`, `INTERVAL`](https://github.com/cockroachdb/cockroach/issues/32472), [`UUID`, `INET`](https://github.com/cockroachdb/cockroach/issues/34417), [`ARRAY`](https://github.com/cockroachdb/cockroach/issues/34420), [`JSONB`](https://github.com/cockroachdb/cockroach/issues/34421), `BIT`, and collated `STRING` are not supported in Avro yet.
+- [`BIT`](bit.html) and [`VARBIT`](bit.html) types are encoded as arrays of 64-bit integers.
+
+    For efficiency, CockroachDB encodes `BIT` and `VARBIT` bitfield types as arrays of 64-bit integers. That is, base 2 `BIT` and `VARBIT` data types are converted to base 10 and stored in arrays. Encoding in CockroachDB is [big-endian](https://en.wikipedia.org/wiki/Endianness), therefore the last value may have many trailing zeroes. For this reason, the first value of each array is the number of bits that are used in the last value of the array.
+
+    For instance, if the bitfield is 129 bits long, there will be 4 integers in the array. The first integer will be `1`; representing the number of bits in the last value, the second integer will be the first 64 bits, the third integer will be bits 65–128, and the last integer will either be `0` or `9223372036854775808` (i.e. the integer with only the first bit set, or `1000000000000000000000000000000000000000000000000000000000000000` when base 2).
+
+    This example is base-10 encoded into an array as follows:
+
+    ~~~
+    {"array": [1, <first 64 bits>, <second 64 bits>, 0 or 9223372036854775808]}
+    ~~~
+
+    For downstream processing, it is necessary to base 2 encode every element in the array (except for the first element). The first number in the array gives you the number of bits to take from the last base 2 number — that is, the most significant bits. So, in the example above this would be `1`. Finally, all the base 2 numbers can be appended together, which will result in the original number of bits, 129.
+
+    To clarify this process, in another example in which you had a bitfield of 135 bits, the array would be as follow when base 10:
```

Comment:
If you're going to have a second example, I think it's a little clearer if the last used digit is a 0, to make it clear that you can't just eliminate all trailing zeroes from the last value. Like if you changed this from 135 bits to 136 bits, so that we're taking 8 bits from the last value and getting `10111110`.

Reply:
Yes, that's a really good point